backend/API/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2 AS base
+WORKDIR /output
+
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build
+WORKDIR /src
+COPY API/ source/
+COPY ./Shared/ Shared/
+WORKDIR source
+RUN dotnet publish -c Release -o output
+
+FROM base AS final
+RUN apt-get update \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4594462)